app/assets/stylesheets/comments.scss

```diff
@@ -44,6 +44,29 @@
     }
   }
 
+  .permalink {
+    @include transition(opacity);
+    opacity: 0;
+
+    [class^="entypo-"],
```

Comment:
You still need to add the `<span class="gray">` around the timestamp+permalink, otherwise the icon has the wrong color in the light themes.

> changing the color instead of the opacity

In your original PR you changed both, the color and the opacity which then had a much stronger difference between hover and non-hover because they doubled up. But let's just do it the same as it is for post permalinks.

Reply:
Yeah sorry I pushed before committing ><